index.ts

```diff
@@ -1048,6 +1057,71 @@ commander
   });
 });
 
+/**
+ * Prints out 5 most recent the StackDriver logs
+ * Use --json for output in json format
+ * Use --open to open logs in StackDriver
+ */
+commander
+  .command('logs')
+  .description('Shows the StackDriver Logs')
+  .option('--json', "Show logs in JSON form")
+  .option('--open', 'Open the StackDriver logs in browser')
+  .action((cmd: LoginOptions) => {
+    function printLogs(entries) {
```

Comment:
Get the first element ([array destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring)):
```ts
function printLogs([entries]) {
```

Requires usage of `printLogs` to be changed (see below).